corehq/apps/reports/commtrack/data_sources.py

```diff
@@ -434,6 +445,8 @@ def get_data(self):
 
             for spoint_id, loc_id in spoint_loc_map.items():
                 loc = locations[loc_id]
+                if loc.is_archived:
+                    continue
```

Comment:
I wanted to switch this source to SQLLocations too, but it uses `Location.path`, which seemed non-trivial to replicate.